client.ts

```diff
@@ -6,16 +6,19 @@ export interface MongoClientConstructorOptions {
   auth: AuthOptions;
   endpoint: string;
   fetch?: typeof fetch;
+  encoding?: 'JSON' | 'EJSON'
```

Comment:
```suggestion
  contentType?: 'ejson' | 'json';
```

it's not really about encoding.

Reply:
I'm still on it...